project/plugin.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("net.databinder.giter8" %% "giter8-plugin" % "0.6.4")
+addSbtPlugin("org.foundweekends.giter8" %% "giter8-plugin" % "0.7.1")
```

Comment:
Yeah I used the new command, pretty cool. I actually forget what this plugin does, maybe for local testing? Didn't take the time to figure out.